CHANGELOG.md

```diff
@@ -21,5 +23,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - initial version of the repo
 - ts-info tool
 
-[Unreleased]: https://github.com/Eyevinn/dash-mpd/compare/v0.1.0...HEAD
-[0.1.0]: https://github.com/Eyevinn/dash-mpd/releases/tag/v0.1.0
+[Unreleased]: https://github.com/Eyevinn/mp2ts-tools/releases/tag/v0.1.1...HEAD
+[0.1.1]: https://github.com/Eyevinn/mp2ts-tools/releases/tag/v0.1.1
```

Comment:
Put changes under unreleased for now, and then we decide on a release later when we have decided exactly what will go into it.